ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -33,6 +33,9 @@ images:
     ubi8-minimal-entitled:
```

Comment:
Should we remove it since we now adding `ubi9-minikube-entiteled`?

Reply:
We can't, we need rehearse jobs to pass.